File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/Orders/Models/OrderLine.cs
using System;

namespace Registration.Orders
{
    public class OrderLine
    {
        public SeatQuantity SeatQuantity { get; private set; }
        public decimal LineTotal { get; private set; }

        public OrderLine() { }
        public OrderLine(SeatQuantity seatQuantity, decimal lineTotal)
        {
            SeatQuantity = seatQuantity;
            LineTotal = lineTotal;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/orders/Models/OrderLine.java
package com.microsoft.conference.registration.domain.orders.Models;

import java.math.BigDecimal;

public class OrderLine {
    public com.microsoft.conference.registration.domain.SeatQuantity SeatQuantity;
    public BigDecimal LineTotal;

    public OrderLine() {
    }

    public OrderLine(com.microsoft.conference.registration.domain.SeatQuantity seatQuantity, BigDecimal lineTotal) {
        SeatQuantity = seatQuantity;
        LineTotal = lineTotal;
    }
}
